packages/gatsby-transformer-remark/src/__tests__/on-node-create.js

```diff
@@ -23,9 +26,13 @@ describe(`Process markdown content correctly`, () => {
     },
   }
 
-  // Make some fake functions its expecting.
-  const loadNodeContent = node => Promise.resolve(node.content)
+  actions = { createNode: jest.fn(), createParentChildLink: jest.fn() }
```

Comment:
can we do:

```
let markdownNode
beforeEach(() => { markdownNode = null})
createNode: jest.fn(markdownNode => createNode = node)
```

to avoid those returns in source? `return {}` is very confusing to read (without looking at tests)